src/bci_build/package/rust.py

```diff
@@ -79,6 +84,7 @@
 # smoke test that gcc works
 RUN gcc --version
 RUN ${{CC}} --version
+COPY {check_fname} /etc/zypp/systemCheck.d/{check_fname}
 """,
```

Comment:
Maybe a `zypper verify` to make sure systemCheck is happy?

Reply:
It doesn't look like `zypper verify` actually checks this:
```
db2920bcc724:/ # echo "requir:ased" > /etc/zypp/systemCheck.d/test.check
db2920bcc724:/ # zypper verify
Refreshing service 'container-suseconnect-zypp'.
Retrieving repository 'SLE_BCI' metadata .........................................[done]
Building repository 'SLE_BCI' cache ..............................................[done]
Loading repository data...
Reading installed packages...
Dependencies of all installed packages are satisfied.
db2920bcc724:/ # echo $?
0
```

Reply:
Probably because of

> echo "requir:ased" > /etc/zypp/systemCheck.d/test.check

Apparently zypper also ignores unknown syntax?

Reply:
Indeed:
```
12a48520765a:/ # echo "requires:ased" > /etc/zypp/systemCheck.d/test.check
12a48520765a:/ # zypper -n verify
Refreshing service 'container-suseconnect-zypp'.
Loading repository data...
Reading installed packages...

Problem: This request will break your system!
  nothing provides the requested 'ased'
Have you enabled all the required repositories?
 Solution 1: ignore the warning of a broken system (requires:ased)

Choose the above solution using '1' or cancel using 'c' [1/c/d/?] (c): c
12a48520765a:/ # echo $?
4
```